### ISSUE TITLE
UV rating text

### ISSUE THREAD

**Issue by kanemari** [bremor/bureau_of_meteorology#27]
Hi there, 

The text for the UV rating appears all in lower case with individual words concatenated. e.g 'Very High' appears as 'veryhigh'.

A small cosmetic problem but just seems out of place next to the Fire Danger rating which shows properly as Low-Moderate, for example. 

Thanks.

**Comment by bremor** [comment]
@DavidFW1960
If I change the state for sensor.location_uv_category_X to something nicer, like "Very High" instead of "veryhigh" is this going to break your lovelace card?

**Comment by DavidFW1960** [comment]
That will be ok. I will just adjust the template sensor anyway which I already use to reformat that.
Personally I'm not a fan of altering anything that is returned by BOM itself in their API and think it's better to use template sensors for this kind of thing.

**Comment by DavidFW1960** [comment]
In templates and weather package:
```

      bom_uv_alert_summary:
        value_template: >
            {% set uv = states('sensor.kariong_uv_category_0') %}
            {% if uv == 'extreme' %} {% set uv = 'Extreme' %}
            {% elif uv == 'veryhigh' %} {% set uv = 'Very High' %} 
            {% elif uv == 'high' %} {% set uv = 'High' %}
            {% elif uv == 'moderate' %} {% set uv = 'Moderate' %}
            {% elif uv == 'low' %} {% set uv = 'Low' %}
            {% endif %}
            {{ uv }}
```
So if you do change this I can either delete that template or modify it

**Comment by bremor** [comment]
> That will be ok. I will just adjust the template sensor anyway which I already use to reformat that.
> Personally I'm not a fan of altering anything that is returned by BOM itself in their API and think it's better to use template sensors for this kind of thing.

I do agree with you, but I also hate creating additional template sensors. I think I will reformat "uv_category" within the integration.

**Comment by DavidFW1960** [comment]
Let me know when you release this and I will make the adjustments.

**Comment by bremor** [comment]
@DavidFW1960 
Fixed in [0.8.3](https://github.com/bremor/bureau_of_meteorology/releases/tag/0.8.3)
